console/src/components/Services/Events/EventTriggers/Modify/OperationEditor.tsx

```diff
@@ -50,6 +51,17 @@ const OperationEditor = (props: OperationEditorProps) => {
     setOperationColumns(existingOpColumns);
   };
 
+  const editorToggle = () => {
+    const allSelected = operationColumns.every;
```

Comment:
@jasvithaMad I think this is incomplete. Can you check/update the implementation?